catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/pinotdbConnection.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/database/pinotdbConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "PinotdbConnection",
+  "description": "Pinotdb Database Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.database.PinotdbConnection",
+  "definitions": {
+    "pinotdbType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": ["Pinotdb"],
+      "default": "Pinotdb"
+    },
+    "pinotdbScheme": {
+      "description": "SQLAlchemy driver scheme options.",
+      "type": "string",
+      "enum": ["pinot"],
```

Comment:
pinotdb also supports the schema `pinot+http` & `pinot+https`, let's add that to the enum as well